orc8r/cloud/go/services/ctraced/obsidian/models/swagger.v1.yml

```diff
@@ -0,0 +1,184 @@
+---
+swagger: '2.0'
+
+magma-gen-meta:
+  go-package: magma/orc8r/cloud/go/services/ctraced/swagger
+  dependencies:
+    - 'orc8r/cloud/go/models/swagger-common.yml'
+  temp-gen-filename: ctraced-swagger.yml
+  output-dir: orc8r/cloud/go/services/ctraced/obsidian
+  types:
+    - go-struct-name: CallTrace
+      filename: call_trace_swaggergen.go
+    - go-struct-name: MutableCallTrace
+      filename: mutable_call_trace_swaggergen.go
+    - go-struct-name: CallTraceConfig
+      filename: call_trace_config_swaggergen.go
+    - go-struct-name: CallTraceState
+      filename: call_trace_state_swaggergen.go
+
+info:
+  title: Call Tracing definitions and paths
+  description: Magma Call Tracing REST APIs
+  version: 1.0.0
+
+basePath: /magma/v1
+
+tags:
+  - name: Call Tracing
+    description: Endpoints related to call tracing
+
+paths:
+  /networks/{network_id}/tracing:
+    get:
+      summary: List all call traces for a network
+      tags:
+        - Call Tracing
+      parameters:
+        - $ref: './orc8r-swagger-common.yml#/parameters/network_id'
+      responses:
+        '200':
+          description: Success
+          schema:
+            type: array
+            items:
+              - $ref: '#/definitions/call_trace'
+        default:
+          $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
+    post:
+      summary: Start a new call trace
+      tags:
+        - Call Tracing
+      parameters:
+        - $ref: './orc8r-swagger-common.yml#/parameters/network_id'
+        - in: body
+          name: Call Trace Configuration
+          description: Configuration of call trace to start
+          required: true
+          schema:
+            $ref: '#/definitions/call_trace_config'
+      responses:
+        '201':
+          description: ID of created call trace
+          schema:
+            type: string
+        default:
+          $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
```

Comment:
Nit: Add newline between this section and  `/networks/{network_id}/tracing/{trace_id}:`